### PR TITLE
Add more unit tests 

### PR DESCRIPTION
**Pull request by KumoLiu**
Fixes # .

### Description

This PR is used to add unit tests on the following bundles:
- [x] endoscopic_tool_segmentation
- [x] pathology_nuclei_segmentation_classification
- [x] wholeBody_ct_segmentation
- [x] pathology_nuclei_classification
- [x] pathology_nuclick_annotation
- [x] lung_nodule_ct_detection

### Status
**Ready/Work in progress/Hold**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

**Review by wyli — APPROVED** [review]
Thanks, it looks good to me. cc @yiheng-wang-nv please help confirm..

**Review by yiheng-wang-nv — APPROVED** [review]

**Comment by yiheng-wang-nv** [comment]
Thanks @KumoLiu , could you also temporarily modify bundles that added unit tests in order to trigger the tests?
Then, revert them back and this PR can merge.

**Comment by yiheng-wang-nv** [comment]
can refer to:
https://github.com/Project-MONAI/model-zoo/pull/455/commits/fcd5a7bde686e4e4137b6ec383aeb1eb46f7d940

**Comment by yiheng-wang-nv** [comment]
need sometime to figure how to solve the resource issue of github action.

**Comment by KumoLiu** [comment]
> need sometime to figure how to solve the resource issue of github action.

I will help take a look at this one.

File: ci/unit_tests/test_endoscopic_tool_segmentation_dist.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    {
+        "bundle_root": "models/endoscopic_tool_segmentation",
+        "train#trainer#max_epochs": 1,
+        "train#dataloader#num_workers": 1,
+        "validate#dataloader#num_workers": 1,
+        "train#deterministic_transforms#3#spatial_size": [32, 32],
+    }
+]
+
+TEST_CASE_2 = [
```
Comment (yiheng-wang-nv) [inline ci/unit_tests/test_endoscopic_tool_segmentation_dist.py:33]: here forgot to disable the load checkpoint handler, thus an error in:
https://github.com/Project-MONAI/model-zoo/actions/runs/6021868190/job/16335497561#step:2:875
Or I suggest to use test_order thing, it's helpful to actually load the model weights (can detect potential error)

Comment (yiheng-wang-nv) [reply ci/unit_tests/test_endoscopic_tool_segmentation_dist.py:33]: Saw your new commits, will resolve this conversation

**Pull request merged**